deb.mk

```diff
@@ -0,0 +1,159 @@
+BUILDSTEP_DESCRIPTION = 'Buildstep uses Docker and Buildpacks to build applications like Heroku'
+BUILDSTEP_REPO_NAME ?= progrium/buildstep
+BUILDSTEP_VERSION ?= 0.0.1
+BUILDSTEP_ARCHITECTURE = amd64
+BUILDSTEP_PACKAGE_NAME = buildstep_$(BUILDSTEP_VERSION)_$(BUILDSTEP_ARCHITECTURE).deb
+
+DOKKU_DESCRIPTION = 'Docker powered mini-Heroku in around 100 lines of Bash'
+DOKKU_REPO_NAME ?= progrium/dokku
+DOKKU_ARCHITECTURE = amd64
+
+PLUGINHOOK_DESCRIPTION = 'Simple dispatcher and protocol for shell-based plugins, an improvement to hook scripts'
+PLUGINHOOK_REPO_NAME ?= progrium/pluginhook
+PLUGINHOOK_VERSION ?= 0.0.1
+PLUGINHOOK_ARCHITECTURE = amd64
+PLUGINHOOK_PACKAGE_NAME = pluginhook_$(PLUGINHOOK_VERSION)_$(PLUGINHOOK_ARCHITECTURE).deb
+
+SSHCOMMAND_DESCRIPTION = 'Turn SSH into a thin client specifically for your app'
+SSHCOMMAND_REPO_NAME ?= progrium/sshcommand
+SSHCOMMAND_VERSION ?= 0.0.1
+SSHCOMMAND_ARCHITECTURE = amd64
+SSHCOMMAND_PACKAGE_NAME = sshcommand_$(SSHCOMMAND_VERSION)_$(SSHCOMMAND_ARCHITECTURE).deb
+
+GEM_ARCHITECTURE = amd64
+
+GOROOT = /usr/lib/go
+GOBIN = /usr/bin/go
+GOPATH = /home/vagrant/gocode
+
+.PHONY: install-from-deb deb-all deb-buildstep deb-dokku deb-gems deb-pluginhook deb-setup deb-sshcommand
+
+install-from-deb:
+	echo "--> Installing docker gpg key"
+	curl --silent https://get.docker.io/gpg 2> /dev/null | apt-key add - 2>&1 >/dev/null
+
+	echo "--> Installing dokku gpg key"
+	curl --silent https://packagecloud.io/gpg.key 2> /dev/null | apt-key add - 2>&1 >/dev/null
+
+	echo "--> Setting up apt repositories"
+	echo "deb http://get.docker.io/ubuntu docker main" > /etc/apt/sources.list.d/docker.list
+	echo "deb https://packagecloud.io/dokku/dokku/ubuntu/ trusty main" > /etc/apt/sources.list.d/dokku.list
+
+	echo "--> Running apt-get update"
+	sudo apt-get update > /dev/null
+
+	echo "--> Installing pre-requisites"
+	sudo apt-get install -y linux-image-extra-`uname -r` apt-transport-https
+
+	echo "--> Installing dokku"
+	sudo apt-get install -y dokku
+
+	echo "--> Done!"
+
+deb-all: deb-buildstep deb-dokku deb-gems deb-pluginhook deb-sshcommand
+	mv /tmp/*.deb .
+	echo "Done"
+
+deb-setup:
+	echo "-> Updating deb repository and installing build requirements"
+	sudo apt-get update > /dev/null
+	sudo apt-get install -qq -y git ruby1.9.1-dev 2>&1 > /dev/null
+	command -v fpm > /dev/null || sudo gem install fpm --no-ri --no-rdoc
+	ssh -o StrictHostKeyChecking=no git@github.com || true
+
+deb-buildstep: deb-setup
+	rm -rf /tmp/tmp /tmp/build $(BUILDSTEP_PACKAGE_NAME)
+	mkdir -p /tmp/tmp /tmp/build
+
+	echo "-> Creating deb files"
+	echo "#!/usr/bin/env bash" >> /tmp/tmp/post-install
+	echo "sleep 5" >> /tmp/tmp/post-install
+	echo "count=\`sudo docker images | grep progrium/buildstep | wc -l\`" >> /tmp/tmp/post-install
```

Comment:
I don't think you need to escape the backticks. It screwed with my syntax highlighting so I removed them and tested. Seems to work both ways.